test/test_jit.py

```diff
@@ -4332,6 +4339,20 @@ def forward(self, x):
 
 
 class TestScript(JitTestCase):
+    def setUp(self):
```

Comment:
i wonder if we could maybe define a subclass of `JitTestCase` that overrides `setUp` anad `tearDown` make `TestScript` and `TestJit` extend it?